src/index.ts

```diff
@@ -150,6 +151,145 @@ app.get("/api/users/:id", async (req, res) => {
   }
 });
 
+
+/**
+ * @openapi
+ * /api/users: 
+ *   delete:
+ *     summary: Removes user
+ *     responses:
+ *       200:
+ *         description: Successfully removed a user
+ *         content:
+ *           text/plain:
+ *             schema:
+ *               type: string
+ *       400:
+ *         description: Id has wrong data type
+ *         content:
+ *           text/plain:
+ *             schema:
+ *               type: string
+ *       404:
+ *         description: No user with given id
+ *         content:
+ *           text/plain:
+ *             schema:
+ *               type: string
```

Comment:
Good description! It's good to specify examples though, it makes it easier to understand the API by potential user.
```suggestion
 *       200:
 *         description: Successfully removed a user
 *         content:
 *           text/plain:
 *             schema:
 *               type: string
  *              example: User with id 1 was deleted
 *       400:
 *         description: Id has wrong data type
 *         content:
 *           text/plain:
 *             schema:
 *               type: string
 *       404:
 *         description: No user with given id
 *         content:
 *           text/plain:
 *             schema:
 *               type: string
```

Reply:
Comment: As long as you respond with the same datatype for the endpoint it's cool - so your usage of `text/plain` is alright here. BUT it also is good to have a uniformity of response type on the API level - so the users can expres that whatever endpoint they use, they'll always get the same data type. So in this case I'd propose to always use json as a response 🙂

Reply:
OK, staying consistent is more important.